### DISCUSSION TITLE
Custom routes for server handlers?

### DISCUSSION THREAD

**Discussion by wjlee2020** [nuxt/nuxt#25168] [Questions] [answered]
Currently, I have set some server handlers in my server/api directory. However, due to some architecture constraints, the backend will take the same /api path.

To mitigate this, I found out that I can have a server/route directory, which will give me the same server handler but the path being something like /myapi for server/route/myapi.ts.

Curious if this is a proper way of doing custom server handler paths (or even if it's okay to do, i'm fine with that too) or if there are others ways.

Again, the issue at the moment: It seems like the backend will take the same url but with /api

Thank you for your support

**Comment by xjccc** [comment, marked as answer]
this is the docs for your problem
[server-routes](https://nuxt.com/docs/guide/directory-structure/server#server-routes)
